tools/metricq-discover.py

```diff
@@ -103,7 +103,7 @@ def on_discover(self, from_token, **kwargs):
             uptime = "a lot of"
 
         try:
-            version = kwargs["version"]
+            version = kwargs["metricqVersion"]
```

Comment:
Please revert that line, I've got my own changes to the discover tool regarding that plus some more

Reply:
Done in 51baa83.